### ISSUE TITLE
ERROR: uvloop does not support Windows at the moment 

### ISSUE THREAD

**Issue by Vincentim27** [aws-rdoty/Amazon-Bedrock-Snowflake-POC#19]
https://github.com/aws-rdoty/Amazon-Bedrock-Snowflake-POC/blob/e9bed0668ec077557e880025219f3259045f5c50/README.md?plain=1#L58
![image](https://github.com/aws-rdoty/Amazon-Bedrock-Snowflake-POC/assets/114856108/5e81b335-6123-4388-99a3-c2ab73615216)

Hi. Can you help me to solve this problem? 
I already use this version uvloop==0.17.0
Thank you.

**Comment by aws-rdoty** [comment]
Hey @Vincentim27 are you using an venv when trying to install the requirements.txt? If so can you try uninstalling uvloop? You can also remove the requirement for uvloop to validate if the application can run without it...

**Comment by Vincentim27** [comment]
Yes I'm using venv when installing requirements.txt. Okay I will try it soon. Thank you for your response @aws-rdoty

**Comment by Vincentim27** [comment]
Hmm, 1 more thing, is that possible that I switch from snowflake to aws? so all of them fully automated on aws. By the way when I try to running this code the deploy works but error like this.
<img width="1440" alt="Screenshot 2024-01-20 at 17 29 35" src="https://github.com/aws-rdoty/Amazon-Bedrock-Snowflake-POC/assets/114856108/e67b3176-4736-4112-ac32-994969d18ffd">
Is that because I didn't change this section?
<img width="613" alt="Screenshot 2024-01-20 at 17 37 14" src="https://github.com/aws-rdoty/Amazon-Bedrock-Snowflake-POC/assets/114856108/1016e666-f602-4f17-a243-dff69ec52c50">
If I need to change, what it gonna be?

Thank you once again.
